text/0080-datetime-extension.md

```diff
@@ -33,7 +33,7 @@ permit(
 when {
   principal.department == "HardwareEngineering" &&
   principal.jobLevel >= 10 &&
-  context.now.timestamp.durationSince(principal.hireDate) > duration("365d")
+  context.now.timestamp.durationSince(principal.hireDate).longerThan(duration("365d"))
```

Comment:
Do we intended to even release a version of the language with this methods? Would we want to keep the methods around after introducing operator overloading? I guess I don't understand the scope the work required to support operator overloading, but I don't like adding functions we don't actually want just to put off some development work.

Reply:
It seems currently the implementation burden is something like this: https://github.com/cedar-policy/cedar/pull/1276/commits/f7c5711103ee493c9770ee93a415d0885e65735e. If it looks reasonable to you, we can just go ahead to implement operator overloading.